Global/MicrosoftOffice.gitignore

```diff
@@ -11,3 +11,6 @@
 
 # PowerPoint temporary
 ~$*.ppt*
+
+# Visio temporary
```

Comment:
Can we make this a bit more clearer? If it is related to autosave then perhaps something like this:

```
# Visio autosave temporary files
```